lib/push-to-s3.js

```diff
@@ -13,37 +12,25 @@ module.exports = function ({
   minify,
   s3bucket
 }) {
-  return function run ([entry, outfile]) {
-    const fileExtension = entry.split('.').pop()
-    if (fileExtension === 'js') {
-      const bundle = browserify({
-        config,
-        entry,
-        env,
-        minify,
-        outfile
-      }).bundle()
-
-      upload({
-        body: bundle,
-        bucket: s3bucket,
-        cloudfront,
-        outfile
-      })
-    } else if (fileExtension === 'css') {
-      transformCss({
-        configPath: config.path,
-        entry,
-        outfile
-      })
-      upload({
-        body: fs.createReadStream(outfile),
-        bucket: s3bucket,
-        cloudfront,
-        outfile
-      })
-    }
-  }
+  console.log('pushing to s3bucket:', s3bucket, '>cloudfront:', cloudfront, '>env:', env, '>minified:', Boolean(minify))
+  return ([entry, outfile]) => (
+    entry.indexOf('.js') !== -1
```

Comment:
Checking the file extension by using `String.indexOf` will not work for certain filenames, ex: `somefile.jsstyle.css` or `somefile.cssstuff.js`.